site/content/integrate/apps/interactivity/_index.md

```diff
@@ -44,49 +47,49 @@ All fields include ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-
 
 Text fields may include:
 
-|  Name        | Type   |   Description                                                                                                                    |         
+| Name         | Type   | Description                                                                                                                      |
 | :----------- | :----- | :------------------------------------------------------------------------------------------------------------------------------- |
-| `subtype`    | string | (Optional) The type of text that will be shown. Available types are one of text, textarea, email, number, password, tel, or url. |
+| `subtype`    | string | (Optional) The type of text that will be shown. Available types are one of input, textarea, email, number, password, tel, or URL. |
 | `min_length` | int    | (Optional) Validate the field length before performing the call.                                                                 |
 | `max_length` | int    | (Optional) Validate the field length before performing the call.                                                                 |
 
 Static select fields include:
 
-|   Name    |  Type   |  Description              |
+| Name      | Type    | Description               |
 | :-------- | :------ | :------------------------ |
 | `options` | Options | Options for the dropdown. |
 
 Each Option includes:
 
-|  Name   | Type   |  Description          |
+| Name    | Type   | Description           |
 | :------ | :----- | :-------------------- |
-| `label` | string | User facing string.   |
-| `value` | string | Machine facing value. |
+| `label` | string | User-facing string.   |
+| `value` | string | Machine-facing value. |
 
 All select also include:
 
-|   Name    | Type |  Description                                                                        |
+| Name      | Type | Description                                                                         |
 | :-------- | :--- | :---------------------------------------------------------------------------------- |
 | `refresh` | bool | (Optional) Allows the form to be refreshed when the value of a dropdown is changed. |
 
-A modal form performs a lookup call to the call endpoint any time a dynamic dropdown is selected. The lookup call will include in the context the app id, the user id, the channel id, and the team id. The values will be populated with the current values of the form. The expected response is error or the following:
+A modal form performs a lookup call to the call endpoint any time a dynamic dropdown is selected. The lookup call includes in the context the app ID, the user ID, the channel ID, and the team ID. The values are populated with the current values of the form. The expected response is error or the following:
```

Comment:
```suggestion
A modal form performs a lookup call to the call endpoint any time a dynamic dropdown is selected. The lookup call will include in the context the app ID, the user ID, the channel ID, and the team ID. The values will be populated with the current values of the form. The expected response is error or the following:
```

Reply:
I realized that my tense change doesn't fit into the documentation as I expected, so I've changed it back.